packages/firebase_ui_localizations/lib/l10n/firebase_ui_fi.arb

```diff
@@ -0,0 +1,537 @@
+  "@verificationEmailSentText": {
+    "description": "Hint text indicating that verification email has been sent",
+    "placeholders": {}
+  },
+  "verificationFailedText": "Emme voineet vahvistaa sähköpostiosoitettasi.",
+  "@verificationFailedText": {
+    "description": "Message indicating that something went wrong during email verification",
+    "placeholders": {}
+  },
+  "resendVerificationEmailButtonLabel": "Lähetä vahvistussähköposti uudelleen",
+  "@resendVerificationEmailButtonLabel": {
+    "description": "Button label that suggests to resend verification email",
+    "placeholders": {}
+  },
+  "verificationEmailSentTextShort": "Vahvistussähköposti lähetetty",
+  "@verificationEmailSentTextShort": {
+    "description": "Short version of the hint text indicating that verification email has been sent",
+    "placeholders": {}
+  },
+  "emailIsNotVerifiedText": "Sähköpostia ei ole vahvistettu",
+  "@emailIsNotVerifiedText": {
+    "description": "Message indicating that email is not verified",
+    "placeholders": {}
+  },
+  "waitingForEmailVerificationText": "Odotetaan sähköpostin vahvistusta",
+  "@waitingForEmailVerificationText": {
+    "description": "Message indicating that email is being verified",
+    "placeholders": {}
+  },
+  "dismissButtonLabel": "Hylkää",
+  "@dismissButtonLabel": {
+    "description": "Dissmiss button label",
+    "placeholders": {}
+  },
+  "okButtonLabel": "OK",
+  "@okButtonLabel": {
+    "description": "OK button label",
+    "placeholders": {}
+  },
+  "checkEmailHintText": "Tarkista sähköpostisi ja napsauta linkkiä vahvistaaksesi sähköpostiosoitteesi.",
+  "@checkEmailHintText": {
+    "description": "Hint text prompting the user to check email for verification link",
+    "placeholders": {}
+  },
+  "doneButtonLabel": "Valmis",
+  "@doneButtonLabel": {
+    "description": "Done button label",
+    "placeholders": {}
+  },
+  "invalidVerificationCodeErrorText": "Antamasi koodi on virheellinen. Yritä uudelleen.",
+  "@invalidVerificationCodeErrorText": {
+    "description": "Error text indicating that entered SMS code is invalid",
+    "placeholders": {}
+  },
+  "ulinkProviderAlertTitle": "Poista kirjautmistavan linkitys",
+  "@ulinkProviderAlertTitle": {
+    "description": "Title that is shown in AlertDialog asking for provider unlink confirmation",
+    "placeholders": {}
+  },
+  "confirmUnlinkButtonLabel": "Poista linkitys",
+  "@confirmUnlinkButtonLabel": {
+    "description": "Unlink confirmation button label",
+    "placeholders": {}
+  },
+  "cancelButtonLabel": "Peru",
+  "@cancelButtonLabel": {
+    "description": "Cancel button label",
+    "placeholders": {}
+  },
+  "unlinkProviderAlertMessage": "Haluatko varmasti poistaa tåmån kirjautmistavan linkityksen",
+  "@unlinkProviderAlertMessage": {
+    "description": "Text that is shown as a message of the AlertDialog confirming provider unlink",
+    "placeholders": {}
+  },
```

Comment:
This doesn't look a valid translation - typos.
Probably should be something like: "Haluatko varmasti poistaa tämän kirjautumistavan linkityksen?"